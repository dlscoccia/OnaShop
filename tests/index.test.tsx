import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import HomePage from '../src/pages/index';

it('renders homepage unchanged', () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
