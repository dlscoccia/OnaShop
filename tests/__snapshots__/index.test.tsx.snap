// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<ShopLayout
  pageDescription="Find the trendy products here!"
  title="Ona Shop - Home"
>
  <Image
    alt="hero banner"
    src={null}
  />
  <ForwardRef(Typography)
    align="center"
    component="h1"
    sx={
      Object {
        "mb": 2,
        "mt": 5,
      }
    }
    variant="h1"
  >
    Choose the perfect outfit!
  </ForwardRef(Typography)>
  <ForwardRef(Typography)
    align="center"
    sx={
      Object {
        "mb": 5,
        "textDecoration": "underline",
      }
    }
    variant="h2"
  >
    All the products you need for your next trip
  </ForwardRef(Typography)>
  <FullScreenLoading />
</ShopLayout>
`;
